fac29/NisaInvest-TFB-FE
# Clean up halimah

## Description
I've changed an image url to an import to work with the deployed version. I've also fixed the type errors re userId in AuthContext (I think) but this will need testing with functions to be certain.

## How Can This Be Tested?
Try using userId from AuthContext instead of hard-coding! And also merge with deployed version to check that the image of Fahan shows up on `/advisors`.

## Checklist:
